wpilibc/shared/src/Commands/InstantCommand.cpp

```diff
@@ -0,0 +1,15 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) FIRST 2011-2016. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "Commands/InstantCommand.h"
+
+InstantCommand::InstantCommand(const std::string& name) : Command(name) {}
+
+InstantCommand::InstantCommand() : Command() {}
```

Comment:
Since only the default base class constructor is called and the constructor body is empty, the header can have `InstantCommand() = default;` and this line can be removed.